src/vitepress/components/VPNavBarLocaleItems.vue

```diff
@@ -35,6 +35,7 @@ const localeLinks = computed(() => {
     <a
       v-if="item.repo"
       :href="item.repo"
+      :title="`${item.text} ${config.i18n?.ariaRepo ?? 'Repository'}`"
```

Comment:
I suggest this part better utilize the `MessageWithLink` interface and be consumed as `ariaRepo.before` and `ariaRepo.after`. And feel free to revamp the interfaces if you have an better idea.